internal/service/cloudbackupsnapshotexportjob/resource_cloud_backup_snapshot_export_job.go

```diff
@@ -57,7 +57,7 @@ func returnCloudBackupSnapshotExportJobSchema() map[string]*schema.Schema {
 		},
 		"custom_data": {
 			Type:     schema.TypeSet,
-			Required: true,
+			Optional: true,
 			ForceNew: true,
```

Comment:
Given that this is now Optional, is the ForceNew still relevant?

Reply:
It is. This resource doesn't have an update method.

Reply:
makes sense